dietstory-routing/login_bonus/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+from django.core.validators import MinValueValidator, MaxValueValidator
+from registration.models import Accounts
+
+MAX_NUM_REWARDS = 20
+
+
+class LoginBonus(models.Model):
+    account = models.OneToOneField(Accounts, on_delete=models.CASCADE, unique=True)
```

Comment:
use Foreign Key instead of OneToOne.

Reply:
I actually speculated on this when researching online before, but will change after more research as I initially started with Foreign key.